pkg/patch/patch.go

```diff
@@ -83,6 +83,18 @@ func createManifestCLI(ctx context.Context, image string, finalTag string, items
 	return cmd.Run()
 }
 
+func createManifestDockerLib(ctx context.Context, image string, finalTag string, items []archDigest) error {
+	ref, _ := reference.ParseNormalizedNamed(image)
+	repo := ref.Name()
+	args := []string{"manifest", "create", "--amend", fmt.Sprintf("%s:%s", repo, finalTag)}
+	for i := range items {
+		args = append(args, items[i].tag)
+	}
+	cmd := exec.CommandContext(ctx, "docker", args...)
```

Comment:
could we do this using the library?

Reply:
Yes, we can. I've now converted it to use `github.com/docker/buildx/util/imagetools` directly